From 417c93af60cc53e70a5f5dcb07d292baaf66f443 Mon Sep 17 00:00:00 2001
From: Geliang Tang <geliang.tang@suse.com>
Date: Tue, 7 Feb 2023 19:41:56 +0800
Subject: [PATCH] Revert "Linux 5.10.167"
Patch-mainline: Never, revert stable version change
References: bsn#83
Modified-by-SEL: No

This reverts commit a5acb54d4066f27e9707af9d93f047f542d5ad88.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 84b78e657357..efdfb40a82ab 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: GPL-2.0
 VERSION = 5
 PATCHLEVEL = 10
-SUBLEVEL = 167
+SUBLEVEL = 166
 EXTRAVERSION =
 NAME = Dare mighty things
 
-- 
2.35.3

